ginivision/src/main/java/net/gini/android/vision/internal/camera/photo/ExifReader.java

```diff
@@ -0,0 +1,63 @@
+package net.gini.android.vision.internal.camera.photo;
+
+import static org.apache.commons.imaging.Imaging.getMetadata;
+
+import android.support.annotation.NonNull;
+import android.support.annotation.Nullable;
+
+import org.apache.commons.imaging.ImageReadException;
+import org.apache.commons.imaging.formats.jpeg.JpegImageMetadata;
+import org.apache.commons.imaging.formats.tiff.TiffField;
+import org.apache.commons.imaging.formats.tiff.constants.ExifTagConstants;
+
+import java.io.IOException;
+import java.util.Arrays;
+
+/**
+ * @exclude
+ */
+class ExifReader {
```

Comment:
This is related to the UUID bugfix, but is also needed for the rotation delta.

We need to read the Exif UserComment to initialize the fields of a Photo instance when created from a Document.